src/store/dashboard/ociDashboard/ociDashboardCommon.ts

```diff
@@ -52,7 +49,7 @@ export function getQueryForWidgetTabs(widget: OciDashboardWidget, filter: OciFil
   };
 
   // Use group_by for service tab and filter for others -- https://github.com/project-koku/koku-ui/issues/846
-  if (widget.currentTab === OciDashboardTab.service_names && widget.tabsFilter && widget.tabsFilter.service_name) {
+  if (widget.currentTab === OciDashboardTab.product_services && widget.tabsFilter && widget.tabsFilter.service_name) {
```

Comment:
We should reference `widget.tabsFilter.product_service` here.

You can remove the old GitHub link from this comment as it was meant for Azure.